pkg/runctl/run_manager.go

```diff
@@ -148,6 +154,7 @@ func (c *runManager) setupServiceAccount(runNamespace string, pipelineCloneSecre
 				serviceAccount.AttachSecrets(pipelineCloneSecretName)
 			}
 			serviceAccount.AttachImagePullSecrets(imagePullSecrets...)
+			serviceAccount.SetDoAutomountServiceAccountToken(false)
```

Comment:
I guess that's what you want
```suggestion
			serviceAccount.SetDoAutomountServiceAccountToken(automountServiceAccountToken)
```

Reply:
I think this is correct. On the service account we want to have this as false by default.

Reply:
In this case it seems the `automountServiceAccountToken` variable is never used and should be removed. Or did I miss something?